react/lifecycle-methods.md

```diff
@@ -1,4 +1,4 @@
-When we're defining a component in React, we know that at least two methods are required: `constructor(props`, and `render()`.
+When we're defining a component in React, we know that at least two methods are required: `constructor(props)`, and `render()`.
```

Comment:
"When defining a stateful component in React, at least two methods are required: `constructor(props)` and `render()`.